packages/runtime-core/src/component.ts

```diff
@@ -148,6 +192,43 @@ export type Component<
   | ConcreteComponent<Props, RawBindings, D, C, M>
   | ComponentPublicInstanceConstructor<Props>
 
+// export type BetterConcreteComponent<
+//   Props extends Record<string, Prop<unknown>> = {},
+//   Emits extends EmitsOptions = {},
+//   Slots = {},
+//   // Allows to expose public API properties, this bypasses Data/Computed/Methods,
+//   // easier to declare manually
+//   Bindings extends Record<string, unknown> = {},
+//   Options extends ComponentOptions = {}
+
+// > =;
+
+export type BetterComponent<
+  Props extends Record<string, unknown> = {},
+  Emits extends EmitsOptions = {},
+  Slots = {},
+  // Allows to expose public API properties, this bypasses Data/Computed/Methods,
+  // easier to declare manually
+  Bindings extends Record<string, unknown> = {},
```

Comment:
Public API fo a component is props, emits, slots and expose. What do you think about `Expose` being 4th param?